[2025-02-17T20:25:07.764+0000] {processor.py:186} INFO - Started process (PID=3197) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:07.765+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:25:07.766+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:25:07.766+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:07.782+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:25:07.780+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:25:07.783+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:07.797+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.037 seconds
[2025-02-17T20:25:38.514+0000] {processor.py:186} INFO - Started process (PID=3259) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:38.515+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:25:38.516+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:25:38.516+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:38.527+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:25:38.525+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:25:38.528+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:25:38.543+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.033 seconds
[2025-02-17T20:26:08.950+0000] {processor.py:186} INFO - Started process (PID=3321) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:08.951+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:26:08.952+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:26:08.952+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:08.966+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:26:08.963+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:26:08.966+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:08.986+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.041 seconds
[2025-02-17T20:26:39.061+0000] {processor.py:186} INFO - Started process (PID=3385) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:39.062+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:26:39.063+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:26:39.062+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:39.074+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:26:39.072+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:26:39.075+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:26:39.090+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.034 seconds
[2025-02-17T20:27:09.528+0000] {processor.py:186} INFO - Started process (PID=3445) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:09.530+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:27:09.531+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:27:09.531+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:09.546+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:27:09.543+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:27:09.548+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:09.565+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.044 seconds
[2025-02-17T20:27:41.027+0000] {processor.py:186} INFO - Started process (PID=3500) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:41.028+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:27:41.029+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:27:41.029+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:41.042+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:27:41.039+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:27:41.044+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:27:41.066+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.044 seconds
[2025-02-17T20:28:11.564+0000] {processor.py:186} INFO - Started process (PID=3562) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:11.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:28:11.566+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:28:11.566+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:11.580+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:28:11.577+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:28:11.581+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:11.602+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.044 seconds
[2025-02-17T20:28:42.059+0000] {processor.py:186} INFO - Started process (PID=3628) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:42.060+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:28:42.061+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:28:42.061+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:42.073+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:28:42.071+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:28:42.074+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:28:42.088+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.034 seconds
[2025-02-17T20:29:12.629+0000] {processor.py:186} INFO - Started process (PID=3693) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:12.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:29:12.631+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:29:12.630+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:12.639+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:29:12.637+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:29:12.640+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:12.656+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:29:43.115+0000] {processor.py:186} INFO - Started process (PID=3758) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:43.116+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:29:43.117+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:29:43.117+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:43.129+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:29:43.127+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:29:43.131+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:29:43.149+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.043 seconds
[2025-02-17T20:30:13.824+0000] {processor.py:186} INFO - Started process (PID=3823) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:13.826+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:30:13.827+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:30:13.827+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:13.847+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:30:13.842+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:30:13.849+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:13.866+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.049 seconds
[2025-02-17T20:30:44.477+0000] {processor.py:186} INFO - Started process (PID=3887) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:44.478+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:30:44.479+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:30:44.479+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:44.492+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:30:44.489+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:30:44.493+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:30:44.510+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.039 seconds
[2025-02-17T20:31:15.199+0000] {processor.py:186} INFO - Started process (PID=3952) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:31:15.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:31:15.201+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:31:15.201+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:31:15.213+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:31:15.211+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:31:15.214+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:31:15.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.037 seconds
[2025-02-17T20:35:12.612+0000] {processor.py:186} INFO - Started process (PID=35) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:12.613+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:35:12.617+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:35:12.616+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:12.629+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:35:12.626+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:35:12.630+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:12.651+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.047 seconds
[2025-02-17T20:35:42.741+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:42.742+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:35:42.744+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:35:42.743+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:42.754+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:35:42.751+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:35:42.755+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:35:42.768+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:36:13.477+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:13.478+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:36:13.480+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:36:13.480+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:13.491+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:36:13.489+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:36:13.492+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:13.505+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.034 seconds
[2025-02-17T20:36:44.355+0000] {processor.py:186} INFO - Started process (PID=230) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:44.355+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:36:44.357+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:36:44.357+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:44.367+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:36:44.365+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:36:44.368+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:36:44.382+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:38:18.575+0000] {processor.py:186} INFO - Started process (PID=35) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:18.576+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:38:18.580+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:38:18.579+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:18.597+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:38:18.594+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:38:18.599+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:18.620+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.052 seconds
[2025-02-17T20:38:48.664+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:48.665+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:38:48.667+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:38:48.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:48.677+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:38:48.674+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:38:48.678+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:38:48.693+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.034 seconds
[2025-02-17T20:39:18.882+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:18.883+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:39:18.885+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:39:18.884+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:18.895+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:39:18.893+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:39:18.896+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:18.914+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.037 seconds
[2025-02-17T20:39:49.356+0000] {processor.py:186} INFO - Started process (PID=230) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:49.357+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:39:49.359+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:39:49.359+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:49.369+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:39:49.367+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:39:49.370+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:39:49.384+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:40:19.596+0000] {processor.py:186} INFO - Started process (PID=295) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:19.596+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:40:19.598+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:40:19.598+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:19.609+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:40:19.607+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:40:19.610+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:19.625+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.034 seconds
[2025-02-17T20:40:49.652+0000] {processor.py:186} INFO - Started process (PID=360) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:49.653+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:40:49.655+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:40:49.654+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:49.663+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:40:49.661+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:40:49.664+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:40:49.678+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.030 seconds
[2025-02-17T20:41:19.752+0000] {processor.py:186} INFO - Started process (PID=425) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:19.753+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:41:19.755+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:41:19.754+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:19.764+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:41:19.762+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:41:19.765+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:19.779+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.033 seconds
[2025-02-17T20:41:50.184+0000] {processor.py:186} INFO - Started process (PID=490) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:50.184+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:41:50.186+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:41:50.186+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:50.195+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:41:50.193+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:41:50.196+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:41:50.210+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:42:20.558+0000] {processor.py:186} INFO - Started process (PID=556) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:42:20.558+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:42:20.560+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:42:20.560+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:42:20.572+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:42:20.569+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:42:20.574+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:42:20.591+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.037 seconds
[2025-02-17T20:43:43.689+0000] {processor.py:186} INFO - Started process (PID=35) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:43:43.690+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:43:43.693+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:43:43.692+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:43:43.705+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:43:43.702+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:43:43.706+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:43:43.722+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.037 seconds
[2025-02-17T20:44:14.311+0000] {processor.py:186} INFO - Started process (PID=100) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:14.312+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:44:14.317+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:44:14.317+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:14.340+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:44:14.336+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:44:14.341+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:14.379+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.073 seconds
[2025-02-17T20:44:44.804+0000] {processor.py:186} INFO - Started process (PID=165) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:44.804+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:44:44.806+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:44:44.806+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:44.817+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:44:44.814+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:44:44.818+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:44:44.832+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.033 seconds
[2025-02-17T20:45:15.492+0000] {processor.py:186} INFO - Started process (PID=230) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:15.493+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:45:15.495+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:45:15.494+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:15.504+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:45:15.502+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:45:15.505+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:15.518+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:45:46.135+0000] {processor.py:186} INFO - Started process (PID=295) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:46.136+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:45:46.138+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:45:46.138+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:46.152+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:45:46.149+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:45:46.153+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:45:46.172+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.044 seconds
[2025-02-17T20:46:16.258+0000] {processor.py:186} INFO - Started process (PID=360) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:16.259+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:46:16.260+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:46:16.260+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:16.269+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:46:16.267+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:46:16.269+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:16.283+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.030 seconds
[2025-02-17T20:46:46.323+0000] {processor.py:186} INFO - Started process (PID=425) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:46.324+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:46:46.325+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:46:46.325+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:46.335+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:46:46.332+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/dag_load_to_stg.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 999, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/dag_load_to_stg.py", line 4, in <module>
    import pymongo
ModuleNotFoundError: No module named 'pymongo'
[2025-02-17T20:46:46.337+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:46:46.351+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.032 seconds
[2025-02-17T20:52:31.050+0000] {processor.py:186} INFO - Started process (PID=58) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:52:31.051+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:52:31.053+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.053+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:52:31.124+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:52:31.315+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.315+0000] {override.py:1912} INFO - Created Permission View: can read on DAG:mongo_to_postgres
[2025-02-17T20:52:31.328+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.327+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG:mongo_to_postgres
[2025-02-17T20:52:31.337+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.336+0000] {override.py:1912} INFO - Created Permission View: can edit on DAG:mongo_to_postgres
[2025-02-17T20:52:31.366+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.365+0000] {override.py:1912} INFO - Created Permission View: can read on DAG Run:mongo_to_postgres
[2025-02-17T20:52:31.378+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.378+0000] {override.py:1912} INFO - Created Permission View: can create on DAG Run:mongo_to_postgres
[2025-02-17T20:52:31.387+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.387+0000] {override.py:1912} INFO - Created Permission View: can delete on DAG Run:mongo_to_postgres
[2025-02-17T20:52:31.393+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.393+0000] {override.py:1912} INFO - Created Permission View: menu access on DAG Run:mongo_to_postgres
[2025-02-17T20:52:31.394+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.393+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:52:31.406+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.405+0000] {dag.py:3262} INFO - Creating ORM DAG for mongo_to_postgres
[2025-02-17T20:52:31.416+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:52:31.416+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:52:31.434+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.389 seconds
[2025-02-17T20:53:01.480+0000] {processor.py:186} INFO - Started process (PID=125) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:01.482+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:53:01.484+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:01.483+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:01.580+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:01.597+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:01.597+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:53:01.618+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:01.618+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:53:01.636+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.161 seconds
[2025-02-17T20:53:31.958+0000] {processor.py:186} INFO - Started process (PID=188) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:31.959+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:53:31.961+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:31.961+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:32.023+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:53:32.039+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:32.038+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:53:32.060+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:53:32.059+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:53:32.081+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.128 seconds
[2025-02-17T20:54:02.137+0000] {processor.py:186} INFO - Started process (PID=253) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:02.138+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:54:02.141+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:02.141+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:02.197+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:02.211+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:02.210+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:54:02.229+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:02.229+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:54:02.246+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.114 seconds
[2025-02-17T20:54:32.707+0000] {processor.py:186} INFO - Started process (PID=318) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:32.708+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:54:32.710+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:32.710+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:32.772+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:54:32.789+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:32.788+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:54:32.810+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:54:32.810+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:54:32.830+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.129 seconds
[2025-02-17T20:55:03.178+0000] {processor.py:186} INFO - Started process (PID=383) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:03.178+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:55:03.180+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:03.180+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:03.236+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:03.251+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:03.250+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:55:03.270+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:03.270+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:55:03.286+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.113 seconds
[2025-02-17T20:55:33.329+0000] {processor.py:186} INFO - Started process (PID=448) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:33.331+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:55:33.333+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:33.333+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:33.408+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:55:33.424+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:33.423+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:55:33.446+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:55:33.445+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:55:33.463+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.139 seconds
[2025-02-17T20:56:04.309+0000] {processor.py:186} INFO - Started process (PID=513) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:04.310+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:56:04.312+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:04.312+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:04.382+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:04.399+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:04.399+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:56:04.420+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:04.420+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:56:04.439+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.138 seconds
[2025-02-17T20:56:35.081+0000] {processor.py:186} INFO - Started process (PID=578) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:35.082+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:56:35.084+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:35.083+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:35.142+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:56:35.156+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:35.156+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:56:35.176+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:56:35.176+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:56:35.195+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.119 seconds
[2025-02-17T20:57:05.318+0000] {processor.py:186} INFO - Started process (PID=644) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:05.318+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:57:05.320+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:05.320+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:05.375+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:05.391+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:05.390+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:57:05.416+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:05.416+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:57:05.431+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.118 seconds
[2025-02-17T20:57:35.826+0000] {processor.py:186} INFO - Started process (PID=709) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:35.827+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:57:35.828+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:35.828+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:35.890+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:57:35.903+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:35.903+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:57:35.927+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:57:35.927+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:57:35.944+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.124 seconds
[2025-02-17T20:58:06.160+0000] {processor.py:186} INFO - Started process (PID=775) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:06.161+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:58:06.163+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:06.163+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:06.226+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:06.239+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:06.239+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:58:06.260+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:06.260+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:58:06.279+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.124 seconds
[2025-02-17T20:58:36.546+0000] {processor.py:186} INFO - Started process (PID=842) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:36.547+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:58:36.548+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:36.548+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:36.603+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:58:36.621+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:36.621+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:58:36.640+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:58:36.640+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:58:36.657+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.116 seconds
[2025-02-17T20:59:06.864+0000] {processor.py:186} INFO - Started process (PID=908) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:06.865+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:59:06.867+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:06.866+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:06.935+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:06.952+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:06.952+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:59:06.973+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:06.973+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:59:06.992+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.134 seconds
[2025-02-17T20:59:37.129+0000] {processor.py:186} INFO - Started process (PID=975) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:37.129+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T20:59:37.131+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:37.131+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:37.190+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T20:59:37.205+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:37.204+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T20:59:37.227+0000] {logging_mixin.py:190} INFO - [2025-02-17T20:59:37.227+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 19:00:00+00:00, run_after=2025-02-17 20:00:00+00:00
[2025-02-17T20:59:37.241+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.118 seconds
[2025-02-17T21:00:07.622+0000] {processor.py:186} INFO - Started process (PID=1040) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:07.623+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:00:07.625+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:07.625+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:07.680+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:07.699+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:07.698+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:00:07.724+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:07.724+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:00:07.742+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.126 seconds
[2025-02-17T21:00:37.991+0000] {processor.py:186} INFO - Started process (PID=1106) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:37.992+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:00:37.996+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:37.996+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:38.087+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:00:38.105+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:38.104+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:00:38.138+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:00:38.138+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:00:38.161+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.176 seconds
[2025-02-17T21:01:08.466+0000] {processor.py:186} INFO - Started process (PID=1172) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:08.467+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:01:08.469+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:08.469+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:08.520+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:08.534+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:08.534+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:01:08.555+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:08.555+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:01:08.569+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.108 seconds
[2025-02-17T21:01:38.821+0000] {processor.py:186} INFO - Started process (PID=1238) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:38.822+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:01:38.824+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:38.824+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:38.884+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:01:38.902+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:38.902+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:01:38.928+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:01:38.927+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:01:38.947+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.133 seconds
[2025-02-17T21:02:09.037+0000] {processor.py:186} INFO - Started process (PID=1304) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:09.038+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:02:09.039+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:09.039+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:09.093+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:09.114+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:09.114+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:02:09.156+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:09.156+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:02:09.173+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.140 seconds
[2025-02-17T21:02:39.575+0000] {processor.py:186} INFO - Started process (PID=1371) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:39.576+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:02:39.579+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:39.578+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:39.648+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:02:39.666+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:39.666+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:02:39.697+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:02:39.697+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:02:39.720+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.150 seconds
[2025-02-17T21:03:09.980+0000] {processor.py:186} INFO - Started process (PID=1438) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:09.981+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:03:09.983+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:09.983+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:10.065+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:10.092+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:10.091+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:03:10.140+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:10.140+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:03:10.166+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.192 seconds
[2025-02-17T21:03:40.247+0000] {processor.py:186} INFO - Started process (PID=1505) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:40.248+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:03:40.250+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:40.250+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:40.304+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:03:40.318+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:40.318+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:03:40.342+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:03:40.342+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:03:40.354+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.112 seconds
[2025-02-17T21:04:10.525+0000] {processor.py:186} INFO - Started process (PID=1569) to work on /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:04:10.526+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/dag_load_to_stg.py for tasks to queue
[2025-02-17T21:04:10.528+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:04:10.528+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:04:10.587+0000] {processor.py:925} INFO - DAG(s) 'mongo_to_postgres' retrieved from /opt/airflow/dags/dag_load_to_stg.py
[2025-02-17T21:04:10.604+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:04:10.604+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2025-02-17T21:04:10.624+0000] {logging_mixin.py:190} INFO - [2025-02-17T21:04:10.624+0000] {dag.py:4180} INFO - Setting next_dagrun for mongo_to_postgres to 2025-02-17 20:00:00+00:00, run_after=2025-02-17 21:00:00+00:00
[2025-02-17T21:04:10.639+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/dag_load_to_stg.py took 0.119 seconds
